docs/migrate.ipynb

```diff
@@ -37,7 +37,7 @@
         "id": "HEYuO5NFwDK9"
       },
       "source": [
-        "# Migrating tf.summary usage to TF 2.0\n",
+        "# Migrating tf.summary usage to TF 2.X\n",
```

Comment:
Could you make this consistent with lower "x" instead? It is written as `2.x` elsewhere.

Reply:
Updated based o your suggestion. Thanks